### ISSUE TITLE
Issue with the Web Scraper

### ISSUE THREAD

**Issue by dd123-a** [soos3d/chat-with-repo-langchain-openai#1]
我对尝试这个项目很感兴趣。然而，我注意到刮纸器似乎出现了故障。

**Comment by soos3d** [comment]
Yes, GitHub changed the structure of the repository pages, so this version does not work anymore. I'll update to a new version soon!
